pubnub-memberships/src/integrationTest/kotlin/com/pubnub/entities/MembershipIntegTest.kt

```diff
@@ -251,15 +253,17 @@ class MembershipIntegTest {
         createSpace(SPACE_ID, SPACE_NAME)
         createSpace(SPACE_ID_02, SPACE_NAME_02)
         val spaceIdWithCustomList: List<Membership.PartialWithSpace> =
```

Comment:
What about changing name of spaceIdWithCustomList to accommodate to new classes?